### PR TITLE
Gear and trinket update

### PR DESCRIPTION
**Pull request by Elivrio**
- Removed PTR links
- Updated Trinket and Gear page

### CODE REVIEW

File: assets/main.scss [hunk 1]
```diff
@@ -121,21 +121,18 @@ div.table-42-25-33 table{
   }
 }
 
-div.table-17-23-20-20-20 table{
+div.table-13-29-29-29 table{
```
Comment (Bloodmallet) [inline assets/main.scss:124]: somehow this style name asynced with its content.

File: guide/general/gear.md [hunk 2]
```diff
@@ -314,7 +318,7 @@ Item | Stats | Location
 
 Item | Stats | Location
 :---: | :---: | :---:
-{{ site.data.gear.erethos_the_empty_promise }} | Vers (67%) > Hast | Scalecommander Sarkareth
+{{ site.data.gear.erethos_the_empty_promise }} | Hast (67%) > Mast | Scalecommander Sarkareth
```
Comment (Bloodmallet) [inline guide/general/gear.md:321]: Is there any specific reason to not write the full stat name?

**Review by Bloodmallet — CHANGES_REQUESTED** [review]

File: guide/general/gear.md (hunk 2 above)
Comment (Elivrio) [reply guide/general/gear.md:321]: Yeah I just wanted the 4 stats to be 4 letters wide. 
No real reason appart from that so I can switch it back if you want.

File: assets/main.scss (hunk 1 above)
Comment (Elivrio) [reply assets/main.scss:124]: Yeah I just updated the value insane without changing the class value, will be fixed

**Review by Bloodmallet — APPROVED** [review]

**Pull request merged**